elementary/clients/dbt/dbt_runner.py

```diff
@@ -241,3 +242,28 @@ def ls(self, select: Optional[str] = None) -> list:
 
     def source_freshness(self):
         self._run_command(command_args=["source", "freshness"])
+
+    def get_installed_packages(self):
+        dbt_packages_folder = os.path.join(os.getcwd(), 'dbt_packages')
```

Comment:
The `dbt_packages` directory will not be created relative to the working directory but rather to the dbt project directory. You can use [the following](https://github.com/elementary-data/elementary/blob/ec17900a8f5712061b32024c600d05c9c9b411fa/elementary/monitor/dbt_project_utils.py#L16) in order to get its location.